docs/reference/commandline/dockerd.md

```diff
@@ -1424,6 +1424,16 @@ This is a full example of the allowed configuration options on Windows:
 }
 ```
 
+#### Feature options
```

Comment:
Looks like something went wrong with a rebase; this part was pulled in from a previous commit